### PR TITLE
task_7 task_8

### PR DESCRIPTION
**Pull request by Pacificus**
Задания по последнему уроку без тестов и функциогальной формы урока6

### CODE REVIEW

**Comment by Pacificus** [comment]
)))

File: task_7/dima.greshchook/Factorial/src/test/java/WrapperForFactorialTest.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import static org.junit.Assert.*;
+
+public class WrapperForFactorialTest {
+
+  @org.junit.Test
```
Comment (parkhomenko) [inline task_7/dima.greshchook/Factorial/src/test/java/WrapperForFactorialTest.java]: Still haven't fixed my comments :(

File: task_7/dima.greshchook/Factorial/src/test/java/WrapperForFactorialTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  public void factorial() throws Exception {
+    WrapperForFactorial wrapperForFactorial = new WrapperForFactorial();
+    //1
```
Comment (parkhomenko) [inline task_7/dima.greshchook/Factorial/src/test/java/WrapperForFactorialTest.java]: Better to split into several test functions

File: task_8/dima.greshchook/PascalTriangle/src/com/dima/Triangle/PascalTriangle.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package com.dima.Triangle;
```
Comment (parkhomenko) [inline task_8/dima.greshchook/PascalTriangle/src/com/dima/Triangle/PascalTriangle.java]: package name shouldn't have capital letters

**Comment by Pacificus** [comment]
Hi!)) Please, check my work.

File: task_9/dima.greshchook/MyArray/src/main/java/dima/com/ArrayMaker.java [hunk 4]
```diff
@@ -0,0 +1,81 @@
+package dima.com;
```
Comment (parkhomenko) [inline task_9/dima.greshchook/MyArray/src/main/java/dima/com/ArrayMaker.java]: Should be com.dima

**Pull request merged**